### DISCUSSION TITLE
MCU boards survey

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp32_8ch_motor_shield#62] [Polls]
The firmware must be supported by esphome.
The board must be a ready to go module (devkit) cause it's very cheaper than soldering everything on a custom pcb.

This is ESP32 devkit, available in 22,86 mm and 25,4 mm width : 
Cost : approx 3.5$
Size : 52 mm x 26 mm
![MFG_ESP32-DEVKITC-VE](https://github.com/user-attachments/assets/4b9387be-397f-4fc7-98ad-a153f38c1844)

This is Esp-c3 supermini : 
Cost : approx 2$
Size : 22.5 x 18mm
![makergo-esp32c3-supermini](https://github.com/user-attachments/assets/8da9a3a1-9b82-48f3-92b1-807e786b1225)

**Comment by nstadigs** [comment]
I don't know enough to vote, but I like these polls! :clap:

**Reply by nliaudat** [reply]
I've updated the survey with MCU description

**Comment by mkaiser** [comment]
Another Suggestion: 

https://www.mouser.de/ProductDetail/Espressif-Systems/ESP32-C6-MINI-1-N4?qs=Jm2GQyTW%2FbjiJX9bVadqSw%3D%3D&_gl=1*pcaiju*_ga*MjUxMzU4MjEwLjE3MjkyNzY2MTI.*_ga_15W4STQT4T*MTcyOTI3NjYxMi4xLjAuMTcyOTI3NjYxMy41OS4wLjA.

3€ and easily solderable. Or jlcpcb can also Assemble IT.

**Reply by nliaudat** [reply]
I want only for devkits as jlcpcb charge 29$ to solder ESP32-C6-MINI-1-N4

**Comment by DerMo92** [comment]
It depends on the available pins Suffizient for the moast bemf. But what about one with Ethernet as Network connection. Or it could be possible to use POE as power supply with a board and use the pins for data connection. For ebusd this is used usr-es1 but I'm not sure about the POE capability.

**Comment by nethawk2** [comment]
Ethernet and Power with the help of PoE is realized on the Olimex ESP32-POE
https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/open-source-hardware
It's expensive compared to other boards, but i like having ethernet instead of WLAN.

![image](https://github.com/user-attachments/assets/4e515bbc-d196-4e31-b702-e21cecb83bea)

**Reply by DerMo92** [reply]
Possible but I think nliaudat wants to make it cheap with standard components and this costs 18€? So will be difficult.

**Reply by nliaudat** [reply]
In fact, you can use any board you want as it runs esphome. 
You have to connect the good wires in the pin headers and you can use the olimex board.

**Reply by IRONROCK20** [reply]
That would be really great!
I'm populating my smart home with these ESP32 POE from Olimex and having the opportunity to use this awesome project on that board is a game changer!

**Comment by grericht** [comment]
esp32c3 has enough pin-outs to realize the whole project?

**Reply by nliaudat** [reply]
Yes, the C3 supermini devkit has enough pins : 

![image](https://github.com/user-attachments/assets/ed73917a-65e2-4251-bcab-3f1589ffa554)

**Reply by DerMo92** [reply]
But this means again two channels on one sensor input. 🤔

**Reply by nliaudat** [reply]
one sensor for 2 channels is good as the update rate of channels is very low.
If the refresh rate of valve position is too fast, you do not increase the accuracy of the heating system, but lower the life of the valve.

**Reply by DerMo92** [reply]
Your right sorry - some wrong connections in my haed. I'm still thinking about the other thread about having 2 valves on one BEMF been stoppt at the same time. Sorry!

**Comment by MarcinJanowski** [comment]
I prefer whole board with soldered goldpins, because i dont line soldering manually.
